### ISSUE TITLE
Community Events Page

### ISSUE THREAD

**Issue by DanVanAtta** [williamdekryger/triplea-game.github.io#14]
I think my recommendation to drop the news section was a bit short sited and biased. A community "events" page where we announce new game releases, new game features, new maps, that all makes sense. What I want to avoid is something like "a new game every month!", and then the newest game is 6 months old because nobody updated the news page recently.

So, community events page? yes? I propose the first news item be:  "new triplea website launched!"

**Comment by williamdekryger** [comment]
You are correct. In fact, I privately had planned to reinstate the news page later, harnessing the full power of Jekyll for blogs. I really just wanted to re-envision it, and maybe bring in members of the community, maybe to write a strategy article, or to showcase new engine features, or whatever. Thinking this should be done before we go live, then?

**Comment by DanVanAtta** [comment]
We're effectively replacing the announcements section of the SF forums. For this feature, i think having a minimal first version before going live probably does make sense. 

We could if we want back date some items with the dates of the previous launches (in fact, having the full game history would be pretty cool)

**Comment by williamdekryger** [comment]
Ok, I can do that. We do have a bit of history that I found in the README that comes with the game [here] (http://williamdekryger.github.io/triplea-game.github.io/about/). I think we can use the changelogs that came with every release to write something about each release.

**Comment by DanVanAtta** [comment]
Maybe it might make sense to post the entire change log. The events page would then have a one or two liner for the event, perhaps:  "Sep 3rd - 1.8.0.9 has been launched. Read about it <a>here</a>"

The events page would also capture map releases and other news too. Posting the change log notes would be nice for people to read while waiting for a game update to download. The change log is included with the game, but you can't exactly read that while waiting for the download. (and of course, once the download is done, a person probably wants to play, not read change notes)
